src/libs/OptionsListUtils.js

```diff
@@ -499,6 +499,8 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
             result.alternateText = showChatPreviewLine && !forcePolicyNamePreview && lastMessageText ? lastMessageText : subtitle;
         } else if (result.isMoneyRequestReport) {
             result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
+        } else if (result.isTaskReport) {
+            result.alternateText = showChatPreviewLine && lastMessageText ? lastMessageTextFromReport : Localize.translate(preferredLocale, 'report.noActivityYet');
```

Comment:
I think you used wrong variable here, which created this issue https://github.com/Expensify/App/issues/33193